kinbot/stationary_pt.py

```diff
@@ -533,8 +537,9 @@ def find_dihedral(self):
         Identify unique rotatable bonds in the structure 
         No rotation around ring bonds and double and triple bonds.
         """
+        
         self.calc_chemid()
```

Comment:
Is this repetitive from the calc_chemid on line 344?

Reply:
I deleted the line on 344 now, because that's preceded by a characterize, which has the chemid calculation. I woud keep the one in 540